src/sentry/utils/kafka.py

```diff
@@ -87,3 +85,34 @@ def handler(signum, frame):
     signal.signal(signal.SIGTERM, handler)
 
     return consumer
+
+
+def _get_legacy_kafka_cluster_options(cluster_name):
+    options = settings.KAFKA_CLUSTERS[cluster_name]
+
+    options = {k: v for k, v in options.items() if k not in ("common", "producers", "consumers")}
+    if options:
+        logger.warning(
+            "You are running with legacy kafka configuration. "
+            "Please check src/sentry/conf/server.py for the new way of configuring kafka"
+        )
```

Comment:
As in snuba, please remove this, it can generate a lot of logs. We will add it back after communicating that this config is deprecated.